src/solvers/dowhy_solver.py

```diff
@@ -162,51 +162,55 @@ def dowhy_solver(
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("--common_data", required=True,
+    parser.add_argument("--common_data",
                         help="Path to common data")
     parser.add_argument(
-        "--verbose", action="store_true", help="Show solver logs"
+       "-v", "--verbose", action="store_true", help="Show solver logs"
+    )
+    parser.add_argument(
+        "-f", "--fast", action="store_true", help="Run in fast mode"
+    )
+    parser.add_argument(
+        "-d", "--debug", action="store_true", help="Debug mode"
     )
-    parser.add_argument("--fast", action="store_true", help="Run in fast mode")
     args = parser.parse_args()
     validator = Validator()
-    data = get_common_data(validator.get_valid_path(args.common_data))
+    if args.common_data is not None:
```

Comment:
Se for None não vai ter a variável 'data', e não vai dar erro lá embaixo?

Reply:
Não, porque lá embaixo ele só usaria a variável data caso não fosse `debug`, quando é `debug` ele não cria `data` nem utiliza `data`

Reply:
Entendi.
Seria mais interessante deixar tudo debaixo do if de debug.
Porque assim você mantém uma consistência. 
Mas tá ok